### PR TITLE
Linking Emergency API with Frontend Screen

### LINKED ISSUE #175: Populate the Emergency Service Description Screen
# Context
Users need to be able to see the full description of the details the Emergency Services Card holds when clicked, a screen will be necessary for that.

# Problem
A screen will be needed for displaying the emergency services, one has already been created, populate that screen

# Solution
A js file for the screen exists, populate it with data retrieved from the API

### PR DESCRIPTION
**Pull request by DemiAdisa**
## Source 
Closes #175 

### Summary of PR 

This PR implements the get request to the Emergency API and parses it into StdCard Components for Viewing. The cards do not have any functionality at the moment, will be working on that next!.

### Purpose 

To include the code for linking the Emergency API and the screen

### Review and Testing Steps

- [ ] Pull this branch
- [ ] Run the server or the docker image of the server (| docker build -t server . | docker run -dp 8080:8080 server)
- [ ] Navigate to app\client\mindjogg\screens\emergency_support\EmergencySupportMainScreen.js and change the IP address to yours in the API URI on line 30 (i.e http://<Your IP Address>:8080/emergency/list)
- [ ] Run the client and navigate to the emergency screen, you should see the cards populated on the screen.

### Additional Information (If applicable)
Nil

### Notes
Currently, a warning about not providing a unique key when I run the map function on an array pops up. I currently have a couple of keys in there (line 82-84) but it still pops up so when you are reviewing this PR, I would like you all to take a look at that and tell me if you know what I am missing. Thank You

### Resources 
For more info, Read the Axios and StdCard Documentation

### CODE REVIEW

**Review by jyaneshseeam — APPROVED** [review]
It works, populating cards work, just build for eslinting fail , please check and update code

**Comment by DemiAdisa** [comment]
> It works, populating cards work, just build for eslinting fail , please check and update code

Cool, did you read the notes part of the PR? Wanted to get some opinions on that

**Comment by jyaneshseeam** [comment]
> > It works, populating cards work, just build for eslinting fail , please check and update code
> 
> Cool, did you read the notes part of the PR? Wanted to get some opinions on that

I saw the notes, I'm not really sure about the mapping function and the keys maybe Andrew or Sontush would be better suited to check those.
For now as long as it works, we may ignore the warnings.

**Comment by DemiAdisa** [comment]
> > > It works, populating cards work, just build for eslinting fail , please check and update code
> > 
> > 
> > Cool, did you read the notes part of the PR? Wanted to get some opinions on that
> 
> I saw the notes, I'm not really sure about the mapping function and the keys maybe Andrew or Sontush would be better suited to check those. For now as long as it works, we may ignore the warnings.

Cool cool 👍

**Comment by sontushkpaul** [comment]
@DemiAdisa Can you remove the space between each component? 
image

What is the purpose of the code ? can you please explain?
image

**Comment by sontushkpaul** [comment]
@DemiAdisa any progress?

**Comment by DemiAdisa** [comment]
Will fix and update this guy today

**Comment by sontushkpaul** [comment]
@DemiAdisa it looks good to me but there are multiple CI/CD fails. please address the issues and have the PR ready. We need to publish it by the end of the sprint

File: app/client/mindjogg/screens/emergency_support/EmergencySupportMainScreen.js [hunk 1]
```diff
@@ -1,99 +1,119 @@
...
-            </TouchableOpacity>
-      </View>
+    /**
+     * Gets a List of Emergency Services
+     */
+    const retrieveServices = async () => {
+      try {
+        //Change the IP address to your Local Address
+        var service = await axios.get(
+          "http://192.168.2.35:8080/emergency/list",
```
Comment (ja-chisholm) [inline app/client/mindjogg/screens/emergency_support/EmergencySupportMainScreen.js:35]: Would you mind putting your IP in a constant? Something like this:
`const BACKEND = "192.168.2.35:8080;
get(BACKEND + "/emergency/list`
Just to make it easier to change in the future

Other than that, this looks good to merge

**Review by ja-chisholm — APPROVED** [review]

File: app/client/mindjogg/screens/emergency_support/EmergencySupportMainScreen.js (hunk 1 above)
Comment (DemiAdisa) [reply app/client/mindjogg/screens/emergency_support/EmergencySupportMainScreen.js:35]: Will attend to now

**Pull request merged**